mmv1/third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -655,6 +655,19 @@ func ResourceBigQueryTable() *schema.Resource {
 							Optional:    true,
 							Description: `When creating an external table, the user can provide a reference file with the table schema. This is enabled for the following formats: AVRO, PARQUET, ORC.`,
 						},
+						"metadata_cache_mode": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Description:  `Metadata Cache Mode for the table. Set this to enable caching of metadata from external data source.`,
+							ValidateFunc: validation.StringInSlice([]string{"AUTOMATIC", "MANUAL"}, false),
+						},
+						"object_metadata": {
+							Type:          schema.TypeString,
+							Optional:      true,
+							Description:   `Object Metadata is used to create Object Tables. Object Tables contain a listing of objects (with their metadata) found at the sourceUris. If ObjectMetadata is set, sourceFormat should be omitted.`,
+							ValidateFunc:  validation.StringInSlice([]string{"SIMPLE"}, false),
```

Comment:
Is there a value in including this field if it only has one valid value? I worry that if we add more values this will fall out of date, and having only the single value possible is really not very useful.

This might work better as a plain string field without the validation func if we do need to configure this

Reply:
removing

Reply:
done